pkg/storage/utils.go

```diff
@@ -16,6 +14,37 @@ type Query struct {
 
 type Modifier func(*Query)
 
+type BalanceOperator string
+
+const (
+	BalanceOperatorE   BalanceOperator = "e"
+	BalanceOperatorGt  BalanceOperator = "gt"
+	BalanceOperatorGte BalanceOperator = "gte"
+	BalanceOperatorLt  BalanceOperator = "lt"
+	BalanceOperatorLte BalanceOperator = "lte"
+)
+
+func (b BalanceOperator) IsValid() bool {
+	switch b {
+	case BalanceOperatorE,
+		BalanceOperatorGt,
+		BalanceOperatorGte,
+		BalanceOperatorLt,
+		BalanceOperatorLte:
+		return true
+	}
+
+	return false
+}
+
+func NewBalanceOperator(s string) (BalanceOperator, bool) {
+	if !BalanceOperator(s).IsValid() {
+		return "", false
+	}
+
+	return BalanceOperator(s), true
+}
+
 func New(qms ...[]Modifier) Query {
```

Comment:
Same reason as above:
```suggestion
func NewQuery(qms ...[]Modifier) Query {
```